tools/hidl/hidlcomp.h

```diff
@@ -989,6 +997,34 @@ class EspMethodInfo
         response_ =strdup(name);
     }
 
+    bool isExecutionProfilingEnabled() const {return enableExecutionProfiling;}
+    void setEnableExecutionProfiling(bool enable)
+    {
+        enableExecutionProfiling = enable;
+
+        if (enable)
```

Comment:
You only call this once, but still prefer to code defensively.  Should check to see if executionProfilingHistogramVariableName is empty befroe appending, or if there were a reason to update then clear it.

Reply:
Changed the method to only turn on profiling (once). There is no reason to support allowing it to be turned off.